tests/framework/CodeInterfaceTests/raven_running_raven_internal_models/test_rom_trainer.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+
```

Comment:
Do we usually have `<TestInfo>` for code interface tests?  I don't know if it makes sense to have it for a sub-input of a RRR test, but maybe it would help with clarity?

Reply:
I see that all the CodeInterface tests have a TestInfo.
But probably the sub-input TestInfo should be removed (since the real test is the master input)